frontend/app/routes/$lang+/_public+/apply+/$id+/type-application.tsx

```diff
@@ -127,7 +130,24 @@ export default function ApplyFlowTypeOfApplication() {
       </div>
       <div className="max-w-prose">
         {errorSummaryItems.length > 0 && <ErrorSummary id={errorSummaryId} errors={errorSummaryItems} />}
-        <p className="mb-6 italic" id="form-instructions">
+        <p className="mb-6">{t('apply:eligibility.type-of-application.page-description')}</p>
+        <h2 className="font-bold">{t('apply:eligibility.type-of-application.apply-self')}</h2>
+        <p>{t('apply:eligibility.type-of-application.apply-self-eligibility')}</p>
+        <ul className="mb-6 list-disc pl-6">
+          <li>{t('apply:eligibility.type-of-application.senior')}</li>
+          <li>{t('apply:eligibility.type-of-application.valid-disability-tax-credit')}</li>
+          <li>{t('apply:eligibility.type-of-application.live-independently')}</li>
+        </ul>
+        <h2 className="font-bold">{t('apply:eligibility.type-of-application.apply-child')}</h2>
+        <p>{t('apply:eligibility.type-of-application.apply-child-eligibility')}</p>
+        <ul className="mb-6 list-disc pl-6">
+          <li>{t('apply:eligibility.type-of-application.sixteen-or-older')}</li>
+          <li>{t('apply:eligibility.type-of-application.under-eighteen')}</li>
+        </ul>
```

Comment:
Add space between headings, paragrahps and lists and use same bullet list style like other pages.

```suggestion
        <div className="mb-6 space-y-4">
        <h2 className="font-bold">{t('apply:eligibility.type-of-application.apply-self')}</h2>
        <p>{t('apply:eligibility.type-of-application.apply-self-eligibility')}</p>
        <ul className="list-disc space-y-1 pl-7">
          <li>{t('apply:eligibility.type-of-application.senior')}</li>
          <li>{t('apply:eligibility.type-of-application.valid-disability-tax-credit')}</li>
          <li>{t('apply:eligibility.type-of-application.live-independently')}</li>
        </ul>
        <h2 className="font-bold">{t('apply:eligibility.type-of-application.apply-child')}</h2>
        <p>{t('apply:eligibility.type-of-application.apply-child-eligibility')}</p>
        <ul className="list-disc space-y-1 pl-7">
          <li>{t('apply:eligibility.type-of-application.sixteen-or-older')}</li>
          <li>{t('apply:eligibility.type-of-application.under-eighteen')}</li>
        </ul>
        </div>
```